resource/ebpf_version.h

```diff
@@ -1,4 +1,4 @@
 // Copyright (c) Microsoft Corporation
 // SPDX-License-Identifier: MIT
 
-#define EBPF_VERSION "0.4.0"
+#define EBPF_VERSION "0.5.0"
```

Comment:
Need to update version numbers in tutorial.md. 
https://github.com/microsoft/ebpf-for-windows/blob/9a9a667064988d50d1b06b48a9b058a49671a0ab/docs/tutorial.md?plain=1#L665

Reply:
Done